src/benchmarklib/benchmark_table_encoder.cpp

```diff
@@ -155,6 +156,8 @@ bool BenchmarkTableEncoder::encode(const std::string& table_name, const std::sha
     });
   }
 
+  generate_chunk_pruning_statistics(table);
```

Comment:
"Other than the file," <-- don't understand what that means, please clarify
"this has "chunk" in it? On purpose?" <-- yes, since those stats created here have the single purpose of being used for Chunk Pruning.

Reply:
I believe the function name mismatche(s/d) the file name